examples/web/index.html

```diff
@@ -283,42 +283,71 @@ <h3 class="typography-heading-small-text mt-1100 mb-800">Stacked blocks</h3>
-          <p><a href="#" class="typography-heading-small-link-underline">Heading Small Link Underline</a></p>
           <p class="typography-heading-small-text">Heading Small Text</p>
-          <p><a href="#" class="typography-heading-xsmall-link-no-underline">Heading XSmall Link No Underline</a></p>
-          <p><a href="#" class="typography-heading-xsmall-link-underline">Heading XSmall Link Underline</a></p>
           <p class="typography-heading-xsmall-text">Heading XSmall Text</p>
-          <p><a href="#" class="typography-body-large-link-no-underline">Body Large Link No Underline</a></p>
-          <p><a href="#" class="typography-body-large-link-underline">Body Large Link Underline</a></p>
           <p class="typography-body-large-text-bold">Body Large Text Bold</p>
           <p class="typography-body-large-text-italic">Body Large Text Italic</p>
           <p class="typography-body-large-text-regular">Body Large Text Regular</p>
           <p class="typography-body-large-button">Body Large Button</p>
-          <p><a href="#" class="typography-body-medium-link-no-underline">Body Medium Link No Underline</a></p>
-          <p><a href="#" class="typography-body-medium-link-underline">Body Medium Link Underline</a></p>
           <p class="typography-body-medium-text-bold">Body Medium Text Bold</p>
           <p class="typography-body-medium-text-italic">Body Medium Text Italic</p>
           <p class="typography-body-medium-text-regular">Body Medium Text Regular</p>
           <p class="typography-body-medium-button">Body Medium Button</p>
-          <p><a href="#" class="typography-body-small-link-no-underline">Body Small Link No Underline</a></p>
-          <p><a href="#" class="typography-body-small-link-underline">Body Small Link Underline</a></p>
           <p class="typography-body-small-text-bold">Body Small Text Bold</p>
           <p class="typography-body-small-text-italic">Body Small Text Italic</p>
           <p class="typography-body-small-text-regular">Body Small Text Regular</p>
           <p class="typography-body-small-button">Body Small Button</p>
-          <p><a href="#" class="typography-body-xsmall-link-no-underline">Body XSmall Link No Underline</a></p>
-          <p><a href="#" class="typography-body-xsmall-link-underline">Body XSmall Link Underline</a></p>
           <p class="typography-body-xsmall-text-regular">Body XSmall Text Regular</p>
+
+          <h3 class="typography-heading-small-text">Links</h3>
+          <p>
+            <a href="#" class="link-primary">Primary Link</a>
+          </p>
+          <p>
+            <a href="#" class="link-secondary">Secondary Link</a>
+          </p>
+          <p class="example-box">
+            <a href="#" class="link-inverted">Inverted Link</a>
+          </p>
+          <p>
+            <a href="#" class="link-primary-disabled">Primary Disabled Link</a>
```

Comment:
Just a question: do you find this better to use than combination of `link-primary link-disabled`? I was thinking of use cases like:

```twig
<a href="#" class="link-primary {% if is_disabled %}link-disabled{% endif %}">
```

vs.


```twig
<a href="#" class="{% if is_disabled %}link-primary-disabled{% else %}link-primary{% endif %}">
```

OK, or:


```twig
<a href="#" class="link-primary{% if is_disabled %}-disabled{% endif %}">
```

I still find the first example more readable though. What do you think?

Reply:
👀 Uhmm, it is probably the most readable one, yea. So I will switch to a separate disabled class. Thanks.